src/libs/AccountUtils.ts

```diff
@@ -5,4 +5,6 @@ import type {Account} from '@src/types/onyx';
 const isValidateCodeFormSubmitting = (account: OnyxEntry<Account>) =>
     !!account?.isLoading && account.loadingForm === (account.requiresTwoFactorAuth ? CONST.FORMS.VALIDATE_TFA_CODE_FORM : CONST.FORMS.VALIDATE_CODE_FORM);
 
-export default {isValidateCodeFormSubmitting};
+const isAccountIDOddNumber = (accountID: number) => accountID % 2 === 1;
```

Comment:
NAB - this could be made a bit more generic as `NumberUtils.isOdd(value: number)`